packages/tools/bundleDefaultPlugins.test.ts

```diff
@@ -0,0 +1,93 @@
+import path = require('path');
+import { downloadPlugins, localPluginsVersion } from './bundleDefaultPlugins';
+import fs = require('fs-extra');
+const fetch = require('node-fetch');
+
+describe('bundlePlugins', function() {
+
+	beforeEach(() => {
+		jest.setTimeout(20000);
+	});
+
+	const testDefaultPluginsIds = {
+		'plugin.calebjohn.rich-markdown': '0.9.0',
+		'io.github.jackgruber.backup': '1.1.0',
+	};
+
+	test('it should get local plugin versions', async () => {
+
+		const testDefaultPluginsIds = {
+			'plugin.calebjohn.rich-markdown': '0.9.0',
+			'io.github.jackgruber.backup': '1.1.0',
+			'plugin-that-dont-exist-locally': '9.9.9',
+		};
+
+		const manifestsPath = path.join(__dirname, '..', '..', '..' , 'joplin/packages/app-cli/tests/services/testPlugins');
+
+		const localPluginsVersions = await localPluginsVersion(manifestsPath, testDefaultPluginsIds);
+
+		expect(localPluginsVersions['io.github.jackgruber.backup']).toBe('1.1.0');
+		expect(localPluginsVersions['plugin.calebjohn.rich-markdown']).toBe('0.9.0');
+		expect(localPluginsVersions['plugin-that-dont-exist-locally']).toBe('0.0.0');
+	});
+
+	test('it should download plugins folder from GitHub with no initial plugins', async () => {
+
+		let manifests = [];
+		try {
+			const manifestData = await fetch('https://raw.githubusercontent.com/joplin/plugins/master/manifests.json');
```

Comment:
Is it actually downloading stuff off GitHub? We can't have this, you'll need to mock this. In general all tests should be able to run offline.